src/lib/formatters/iac-output/v2/issues-list/issue.ts

```diff
@@ -13,7 +13,13 @@ export function formatIssue(result: FormattedOutputResult): string {
 
   const propertiesOutput = formatProperties(result);
 
-  return titleOutput + EOL + propertiesOutput;
+  return (
```

Comment:
nit: Refactor suggestion for the `formatIssue` function, for organizing the addition of the prefix padding to all output components of an issue.

```suggestion
export function formatIssue(result: FormattedOutputResult): string {
  const issueOutputComponents: string[] = [
    formatTitle(result.issue),
    ...formatProperties(result),
  ];

  return issueOutputComponents
    .map((issueOutputComponent) => contentPadding + issueOutputComponent)
    .join(EOL);
}
```

Reply:
Thanks for the suggestion. I find this format a bit harder to read personally. If you prefer this then feel free to open a refactoring PR after my PR